doc/apiv3-documentation.apib

```diff
@@ -3155,6 +3233,7 @@ Note that due to sharing this might be more than the versions *defined* by that
 | category           | The category of the work package                                                                                                                      | Category     |              | READ / WRITE          |                                           |
 | priority           | The priority of the work package                                                                                                                      | Priority     | not null     | READ / WRITE          |                                           |
 | project            | The project to which the work package belongs                                                                                                         | Project      | not null     | READ                  |                                           |
+| revisions          | Revisions that are referencing the work package                                                                                                       | Revision     |              | READ                  | **Permission** view changesets            |
```

Comment:
Ah, no - I was so busy with the endpoint that I forgot to fully describe it here.